src/structures/VoiceState.js

```diff
@@ -58,7 +58,7 @@ class VoiceState extends Base {
      */
     this.sessionId = data.session_id ?? null;
     /**
-     * Whether this member is streaming using "Go Live"
+     * Whether this member is streaming using "Screen Share"
```

Comment:
Pretty sure the feature is still called "Go Live"..did that change?

Reply:
Yep, I checked it myself